Automattic/newspack-newsletters
# fix: prevent duplicate js plugin registration

### All Submissions:






### Changes proposed in this Pull Request:

Prevents duplicate registration of the `newspack-newsletters-ads-settings` editor plugin.

### How to test the changes in this Pull Request:

1. While in the master branch, draft a new newsletter and confirm you get `Plugin "newspack-newsletters-ads-settings" is already registered.` error in the console
2. Check out this branch, refresh, and confirm the error is gone
3. Make sure you are still able to change settings in the "Ads Settings" panel

### Other information:
